src/routes/admin/_admin.tsx

```diff
@@ -130,9 +118,11 @@ function LayoutComponent() {
           <Sheet>
             <h1>SPV Wallet Admin</h1>
           </Sheet>
-          <PageRefreshButton />
-          <ModeToggle />
-          <Profile />
+          <div className="ml-auto flex items-center gap-4">
+            <PageRefreshButton />
+            <ModeToggle />
+            <Profile />
+          </div>
```

Comment:
As far as I understand - by the way of "adjusting to new js-client" you've decided to add small improvements to UI?
- That's fine 👍 

I'm just curious, what this ☝️ modification fixes?

Reply:
I'll be honest, I forgot to delete if from when I was trying to add a place for a acc balance but It will be added nontheless. It is basically a same thing that was already existing but currently on the "nav bar" we have a logo and refresh button along with theme settings and profile. There was a class 'ml-auto' added to refresh button so this approach from my understaning was: we add a new thing, we remove the margin from the most left item and add it to the other most left item. This way we don't need to bother with that and all the items that need to be on the right side grouped will automatically be aligned right